src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String CSV_SEPARATOR = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final byte OP_TYPE_INDEX = 0;
+    private static final byte AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Please, don't put all your logic to a single method. Divide it into several methods, which are responsible for only 1 task. Try to follow "single responsibility principle"
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
    }
```